libs/components/tabs/src/lib/modules/tabs/tab-button.component.scss

```diff
@@ -66,7 +50,7 @@
         var(--sky-comp-button-borderless-space-inset-left)
     );
 
-    // when v1 modern is dropped, investigate adding icon borderless button classes to close element
+    // investigate adding icon borderless button classes to close element
```

Comment:
if we can't address it now, can we leave the "when v1 dropped" and make a story to go investigate all of these potential improvements in the future? I think keeping the v1 context will make it easier to find later